tutorials/dvariate.py

```diff
@@ -0,0 +1,60 @@
+r"""
+D-Variate Independence Testing
+*********************
```

Comment:
Tutorial not showing up on the website. If I had the guess, it is because the line of stars is not long enough